spring-boot/admin/src/main/resources/templates/admin/fragments/proxies/list.html

```diff
@@ -137,8 +137,16 @@ <h2 th:text="#{admin.proxies.list.title}">[Proxies Administration]</h2>
                                     </td>
                                     <td th:text=" ${proxy.description}">proxy description</td>
                                     <td class="text-center ">
-                                        <a th:href="|@{/admin/proxies/}${proxy.id}/download|"
-                                           class="glyphicon glyphicon-download-alt"/>
+                                        <a th:href="|@{/admin/proxies/}${proxy.id}/download|">
+                                           <button>
+                                               <i class="glyphicon glyphicon-download-alt"></i>
+                                           </button>
+                                        </a>
+                                        <a th:href="|@{/admin/proxy-docker-config/}${proxy.id}/download|">
```

Comment:
To be in line with the other download link, it would be better to use something along the lines of:
```suggestion
                                        <a th:href="|@{/admin/proxies/}${proxy.id}/downloadDockerComposeFile|">
```

Reply:
Done.